Network/Socket/Types.hsc

```diff
@@ -191,6 +184,8 @@ packSocketType' stype = case Just stype of
 #endif
     _ -> Nothing
 
+{-# DEPRECATED packSocketType "Don't use this" #-}
```

Comment:
Could we get some reasoning in place for these "Don't use this" messages? Why shouldn't we use them?

Reply:
Done.

Reply:
Could you clean up the other instances of "Don't use this"?

Reply:
OK. I changed three "Don't use this".